_posts/2019-03-06-release-1.8.0.md

```diff
@@ -0,0 +1,236 @@
+  ([FLINK-5697](https://issues.apache.org/jira/browse/FLINK-5697))**: The Kinesis
+  Consumer can now emit periodic watermarks that are derived from per-shard watermarks,
+  for correct event time processing with subtasks that consume multiple Kinesis shards.
+
+* **New consumer for DynamoDB Streams to capture table changes
+  ([FLINK-4582](https://issues.apache.org/jira/browse/FLINK-4582))**: `FlinkDynamoDBStreamsConsumer`
+  is a variant of the Kinesis consumer that supports retrieval of CDC-like streams from DynamoDB tables.
+
+* **Support for global aggregates for subtask coordination
+  ([FLINK-10887](https://issues.apache.org/jira/browse/FLINK-10887))**:
+  Designed as solution for global source watermark tracking, `GlobalAggregateManager`
+  allows sharing of information between parallel subtasks. This feature will
+  be integrated into streaming connectors for watermark synchronization and
+  can be used for other purposes with a user defined aggregator.
+
+## Important Changes
+
+* **Changes to bundling of Hadoop libraries with Flink
+  ([FLINK-11266](https://issues.apache.org/jira/browse/FLINK-11266))**:
+  Convenience binaries that include hadoop are no longer released.
+
+  If a deployment relies on `flink-shaded-hadoop2` being included in
+  `flink-dist`, then you must manually download a pre-packaged Hadoop
+  jar from the optional components section of the [download
+  page](https://flink.apache.org/downloads.html) and copy it into the
+  `/lib` directory.  Alternatively, a Flink distribution that includes
+  hadoop can be built by packaging `flink-dist` and activating the
+  `include-hadoop` maven profile.
+
+  As hadoop is no longer included in `flink-dist` by default, specifying
+  `-DwithoutHadoop` when packaging `flink-dist` no longer impacts the build.
+
+* **FlinkKafkaConsumer will now filter restored partitions based on topic
+  specification
+  ([FLINK-10342](https://issues.apache.org/jira/browse/FLINK-10342))**:
+  Starting from Flink 1.8.0, the `FlinkKafkaConsumer` now always filters out
+  restored partitions that are no longer associated with a specified topic to
+  subscribe to in the restored execution. This behaviour did not exist in
+  previous versions of the `FlinkKafkaConsumer`. If you wish to retain the
+  previous behaviour, please use the
+  `disableFilterRestoredPartitionsWithSubscribedTopics()` configuration method
+  on the `FlinkKafkaConsumer`.
+
+  Consider this example: if you had a Kafka Consumer that was
+  consuming from topic `A`, you did a savepoint, then changed your
+  Kafka consumer to instead consume from topic `B`, and then restarted
+  your job from the savepoint. Before this change, your consumer would
+  now consume from both topic `A` and `B` because it was stored in
+  state that the consumer was consuming from topic `A`. With the
+  change, your consumer would only consume from topic `B` after
+  restore because we filter the topics that are stored in state using
```

Comment:
restore because **it now** **filters** the topics that are stored in state using